openedx/core/djangolib/nose.py

```diff
@@ -13,9 +13,6 @@ def setup_databases(self):
         """ Setup databases and then flush to remove data added by migrations. """
         return_value = super(NoseTestSuiteRunner, self).setup_databases()
 
-        # Delete all data added by data migrations. Unit tests should setup their own data using factories.
-        call_command('flush', verbosity=0, interactive=False, load_initial_data=False)
```

Comment:
It was part of the initial commit when Usman added skipping migrations via env setting. It doesn't look like it was created to address any specific bug, so I'm interpreting it as a defensive measure. Hmmm... actually, I should just put it behind a conditional, since it's definitely safe to do if you skip migrations, which is the most common case in dev anyway.